data/product.d/centos.conf

```diff
@@ -4,29 +4,4 @@
 product_name = CentOS Linux
 
 [Base Product]
-product_name = Red Hat Enterprise Linux
```

Comment:
I am not sure if we want to complicate dependencies between configurations. CentOS Stream could be just an alias for CentOS Linux. Therefore, I would prefer Red Hat Enterprise Linux -> CentOS Linux -> CentOS Stream. @jkonecny12, what do you think?

Reply:
With CentOS Stream ahead of CentOS Linux, if one was to become an alias of the other, I'd prefer CentOS Stream to be the 'real' release with CentOS Linux as the alias.

Reply:
If we want to be completely correct here than it should be ``CentOS Stream`` -> ``RHEL`` -> ``CentOS`` but I don't think we want to complicate things so much.

For that reason I agree with @poncovka . We won't be reflecting how things really happen so or so. But if we go path
``RHEL`` -> ``CentOS`` -> ``CentOS Stream`` then we would make something like stability order. I mean most stable systems on the left and the most prone to change on the right. Do you agree @jcpunk ?

Reply:
What should probably be the way this works is that there's a base CentOS config that both CentOS Linux and CentOS Stream derive from, since there's a lot that's in common between the two.

Reply:
This is something I'll want to think through a bit... with the holiday coming up it can certainly sit for a time.

Reply:
In light of today's announcement I'd like to revisit this.

Reply:
IMO the two centoses already were related mostly by name and the announcement means any divergences will only grow. So the correct approach would be to derive both from RHEL, which is the real centerpiece anyway. @Conan-Kudo's idea expands on that, and sounds almost good, except I wouldn't go as far as creating abstract classes in config files...

Just one more opinion.

Reply:
Let's use @jcpunk's solution.